doc/classes/CodeEdit.xml

```diff
@@ -590,6 +590,12 @@
 		<member name="text_direction" type="int" setter="set_text_direction" getter="get_text_direction" overrides="TextEdit" enum="Control.TextDirection" default="1" />
 	</members>
 	<signals>
+		<signal name="new_line_added">
```

Comment:
Indeed, this signal's description should be moved between `code_completion_requested` and `symbol_lookup` to keep alphabetical order.

Reply:
Fixed with the latest commits